hyperwallet/utils/encryption.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+import os
+import json
+import requests
+import time
+import sys
+
+from jwcrypto import jwk, jws as cryptoJWS, jwe
+from jwcrypto.common import json_encode, json_decode
+from jwcrypto.common import base64url_decode, base64url_encode
+from jose import jws
+
+from hyperwallet.exceptions import HyperwalletException
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
+
+
+class Encryption(object):
+    '''
+    The Hyperwallet API Client.
+
+    :param clientPrivateKeySetLocation:
+        The location(url or path to file) of client's private JWK key set. **REQUIRED**
+    :param hyperwalletKeySetLocation:
+        The location(url or path to file) of hyperwallet public JWK key set. **REQUIRED**
+    :param encryptionAlgorithm:
+        JWE encryption algorithm.
+    :param signAlgorithm:
+        JWS signature algorithm.
+    :param encryptionMethod:
+        JWE body encryption method.
+    :param jwsExpirationMinutes:
+        Time in minutes when JWS signature is valid after creation.
+    '''
+
+    def __init__(self,
```

Comment:
if you execute **Make test** then you can see some code style warning like this: 
**hyperwallet/utils/encryption.py:37:13: E128 continuation line under-indented for visual indent**

Reply:
Fixed